common-service-core/src/main/java/com/ca/mfaas/security/SecurityUtils.java

```diff
@@ -230,7 +231,7 @@ public static KeyPair generateKeyPair(String algorithm, int keySize) {
             kpg.initialize(keySize);
             kp = kpg.generateKeyPair();
         } catch (NoSuchAlgorithmException e) {
-            log.error(e.getMessage(), e);
+            log.debug("An error occurred while generating keypair");
```

Comment:
Maybe add stacktrace of error or original error message at least in TRACE level? Because it will help to investigate the root of problem.

Reply:
Added the original error message